addons/mail/static/src/core/common/chat_window.scss

```diff
@@ -1,7 +1,7 @@
 .o-mail-ChatWindow {
     width: $o-mail-ChatWindow-width;
 
-    z-index: $zindex-dropdown + 1;
+    z-index: $zindex-sticky;
```

Comment:
Shouldn't we have a "max" between the 2 values instead?
So we keep all the references above which it should be, in case either of them changes.

Reply:
`zindex-dropdown`, `zindex-sticky`, etc. are defined in a central place and they have increasing values:
https://github.com/odoo/odoo/blob/saas-18.1/addons/web/static/lib/bootstrap/scss/_variables.scss#L1134

They act as threshold, so not worth making rules more complex by pretending they are differently managed.